src/component-library/LoadingSpinner/LoadingSpinner.tsx

```diff
@@ -0,0 +1,10 @@
+import { BaseLoadingSpinner, BaseLoadingSpinnerProps } from './LoadingSpinner.style';
+
+type LoadingSpinnerProps = BaseLoadingSpinnerProps & React.ComponentPropsWithRef<'span'>;
+
+const LoadingSpinner = ({ diameter = 48, thickness = 10, ...rest }: LoadingSpinnerProps): JSX.Element => {
+  return <BaseLoadingSpinner diameter={diameter} thickness={thickness} {...rest} />;
```

Comment:
Do we need the return statement or is a syntax preference?

Reply:
We may not need return statement but I guess we do not have any restrictions on that.

Reply:
Good point Anton - we're not consistent on this. I prefer to omit return statements when they're not needed, but arguably it would be easier to enforce a convention of always using return statements. I could be wrong, but I don't think this is something we can enforce with tooling either which would be my preference :) 

For now I think this is fine as it is - we can make a decision outside of this PR, and then update all the component library components afterwards. There aren't many of them yet, so it's a trivial change.